Application/Tests/Stubs/StubDetailPresenter.swift

```diff
@@ -11,6 +11,10 @@ class StubDetailPresenter: StubSelectionPresenter {
 
 extension StubDetailPresenter: DetailPresenter {
 
+    func foodTablePresentation(of viewModel: FoodTableViewModel) -> DismissablePresentation {
+        return DismissablePresentation.stub()
+    }
```

Comment:
Had to include this because `StubDetailPresenter` can't also inherit from `StubFoodTablePresenter` in addition to `StubSelectionPresenter`. They are both classes and not protocols. Any thoughts or ideas on this?

Reply:
(Maybe fine because it's just for testing?)

Reply:
Ah, yeah... This should probably either be done with protocol extensions instead (or just implemented uniquely in every stub) rather than using class inheritance. What you have here is fine for this PR if you'd prefer to punt on that.